Cutelyst/Plugins/Utils/Validator/validator.cpp

```diff
@@ -199,9 +203,9 @@ void Validator::fillStash(Context *c)
         c->setStash(QStringLiteral("validationErrorStrings"), validationErrorStrings);
         c->setStash(QStringLiteral("validationErrorFields"), validationErrorFields);
 
-        if (!d->params.isEmpty()) {
-            QMap<QString,QString>::const_iterator i = d->params.constBegin();
-            while (i != d->params.constEnd()) {
+        if (!params.isEmpty()) {
```

Comment:
In normal operation is params usually empty? If not avoid the test as the iterator loop won't run if empty and you will be doing the test twice.